src/OutputWriters/jld2_output_writer.jl

```diff
@@ -113,7 +113,7 @@ Write out 3D fields for w and T and a horizontal average:
 using Oceananigans, Oceananigans.OutputWriters, Oceananigans.Fields
 using Oceananigans.Utils: hour, minute
 
-model = NonhydrostaticModel(grid=RectilinearGrid(size=(1, 1, 1), extent=(1, 1, 1)))
+model = NonhydrostaticModel(grid=RectilinearGrid(size=(1, 1, 1), extent=(1, 1, 1)), tracers=(:T, :S))
```

Comment:
Why do we need two tracers here? Does this has anything to do with the demonstration regarding jld2 output? If so, OK! If not perhaps one or no tracer would do the job?